index.js

```diff
@@ -69,30 +69,30 @@ var appServer = function(config) {
     // set up a router to hang all alexa apps off of
     var alexaRouter = express.Router();
 
-    var normalizedRoot = root.indexOf('/') === 0 ? root : '/' + root;
+    var normalizedRoot = utils.normalizeApiPath(root);
     self.express.use(normalizedRoot, alexaRouter);
 
     var app_directories = function(srcpath) {
       return fs.readdirSync(srcpath).filter(function(file) {
-        return fs.statSync(path.join(srcpath, file)).isDirectory();
+        return utils.isValidDirectory(path.join(srcpath, file));
       });
     };
 
     app_directories(app_dir).forEach(function(dir) {
-      var package_json = path.join(app_dir, dir, "/package.json");
-      if (!fs.existsSync(package_json) || !fs.statSync(package_json).isFile()) {
+      var package_json = path.join(app_dir, dir, "package.json");
+      if (!utils.isValidFile(package_json)) {
         self.error("   package.json not found in directory " + dir);
         return;
       }
 
-      var pkg = JSON.parse(fs.readFileSync(package_json, 'utf8'));
+      var pkg = utils.readJsonFile(package_json);
       if (!pkg || !pkg.main || !pkg.name) {
         self.error("   failed to load " + package_json);
         return;
       }
 
-      var main = fs.realpathSync(path.join(app_dir, dir, pkg.main));
-      if (!fs.existsSync(main) || !fs.statSync(main).isFile()) {
+      var main = path.join(__dirname, app_dir, dir, pkg.main);
```

Comment:
Yea, I did do some testing and it is kind of a breaking change. I'll open a new issue in the meantime.